src/frontend/components/TimedTextListItem/TimedTextListItem.tsx

```diff
@@ -147,7 +153,11 @@ export class TimedTextListItem extends React.Component<
           />
           &nbsp;/&nbsp;
           <Link to={UPLOAD_FORM_ROUTE(modelName.TIMEDTEXTTRACKS, track.id)}>
-            <FormattedMessage {...messages.replace} />
+            <FormattedMessage
+              {...(track.upload_state === uploadState.PENDING
```

Comment:
I'm ok with this but the upload_state can never be PENDING while we're showing this, right now

Reply:
If you don't upload the file and go back to the dashboard, the track is in `PENDING` state. This is what I saw this morning.

Reply:
Oh right, it's an edge case I did not think about. Nice catch!